docs/helm-chart/production-guide.rst

```diff
@@ -415,18 +415,17 @@ Built-in secrets and environment variables
 ------------------------------------------
 
 The Helm Chart by default uses Kubernetes Secrets to store secrets that are needed by Airflow.
-Content of those secrets is by default turned into environment variables that are read by
+The contents of those secrets are by default turned into environment variables that are read by
 Airflow (some of the environment variables have several variants to support older versions of Airflow).
 
-By default, names of the secret used is determined from the Release Name used when Helm Chart is deployed,
-but you can also set your own names for the secrets to override the variables or disable using the secrets
+By default, the secret names are determined from the Release Name used when the Helm Chart is deployed,
+but you can also use a different secret to set the variables or disable using secrets
 entirely and rely on environment variables (specifically if you want to use ``_CMD`` or ``__SECRET`` variant
 of the environment variable.
 
 However, Airflow supports other variants of setting secret configuration - you can specify a system
 command to retrieve and automatically rotate the secret (by defining variable with ``_CMD`` suffix) or
-to retrieve a variable from secret backed (by defining the variable with ``_SECRET`` suffix). All
-``AIRFLOW__*`` variables implement those patterns.
```

Comment:
I removed this as only a select few `AIRFLOW__*`'s actually support `_CMD` and `_SECRET`.

Reply:
Yep.